src/overlays/actors/ovl_Boss_Mo/z_boss_mo.c

```diff
@@ -570,25 +542,33 @@ void BossMo_Tentacle(BossMo* this, PlayState* play) {
                     this->timers[0] = 60;
                 }
             }
-            if (this->timers[0] > 50) {
-                rippleCount = 1;
-            } else if (this->timers[0] > 40) {
-                rippleCount = 3;
-            } else if (this->timers[0] > 30) {
-                rippleCount = 5;
-            } else if (this->timers[0] > 20) {
-                rippleCount = 8;
-            } else {
-                rippleCount = 3;
-            }
-            for (indS1 = 0; indS1 < rippleCount; indS1++) {
-                randFloat = Rand_ZeroFloat(50.0f);
-                randAngle = Rand_ZeroFloat(0x10000);
-                ripplePos = this->actor.world.pos;
-                ripplePos.x += sinf(randAngle) * randFloat;
-                ripplePos.z += cosf(randAngle) * randFloat;
-                ripplePos.y = MO_WATER_LEVEL(play);
-                BossMo_SpawnRipple(play->specialEffects, &ripplePos, 40.0f, 110.0f, 80, 290, MO_FX_SMALL_RIPPLE);
+            if (1) {
```

Comment:
More so out of my own curiosity, is the `if (1)` needed? Do we have a preference of having an `if (1) {` to start a new block for variable declaration, or just starting with a `{`?

Reply:
we've been doing it when its needed and also when scoped temps are needed, since it knocks two birds out with one stone. whether or not its needed in this case im not sure

Reply:
I've been preferring a plain `{`, but in this case `if (1) {` is needed. I'm not sure why, or if there's another way to write it